### DISCUSSION TITLE
Problem in WatchFace Maker to create monthly calendar

### DISCUSSION THREAD

**Discussion by ricardfs** [zepp-health/zeppos-forum#55] [Developer Feedback]
I have a small problem in Watch Face Maker that I cannot solve, it is related to the creation of Watch Faces for the Amazfit Bip series.

I am trying to create a calendar that shows the entire month we are in, that is, one image per month with all the days of the month that occupies half of the screen.

I have the correct images for each month, I have tested the GTS2 model and the images load without any problem and this model is quite compatible with Bip U, for example.

Well, the problem I have is the following:

I go to the Time menu and choose the date element, in the month element I choose the image option to upload the 12 images that will make up the complete year, in principle the images upload without a problem but they do not appear in the preview:

![screenshot-watchface zepp com-2023 01 19-18_27_10](https://user-images.githubusercontent.com/5721169/213521120-c579d37f-e933-40b0-b4ad-1cc94c94fb59.png)

Instead only 3 appear white pixels, I have tried different images and none of them appear in the preview, if I use the same images with the number option they do appear correctly but of course in this option it only counts 10 months and I need 12.

![screenshot-watchface zepp com-2023 01 19-18_24_43](https://user-images.githubusercontent.com/5721169/213521352-ccd53b65-1fc3-418b-8e76-c15452470221.png)

Any idea what might be happening? What seems curious to me is that it only happens with the Bip model series.

**Comment by MyoungXUE** [comment]
I'm sorry for the long wait due to Chinese New Year, I've already forwarded the question to the relevant colleagues, please be patient and wait for our reply.

**Comment by ricardfs** [comment]
I am waiting, thank you for your response.
